apps/site/astro.config.ts

```diff
@@ -36,20 +36,29 @@ export default defineConfig({
 			customCss: ['./src/styles/starlight.css'],
 			favicon: '/favicon.png',
 			components: {
-				Header: './src/components/StarlightHeader.astro',
+				Header: './src/components/starlight/StarlightHeader.astro',
+				Sidebar: './src/components/starlight/StarlightSidebar.astro',
 			},
 		}),
 		tailwind({
 			applyBaseStyles: false,
 		}),
 		icon({
 			include: {
-				ri: ['github-line', 'twitter-x-line', 'discord-line'],
+				ri: [
+					'github-line',
+					'twitter-x-line',
+					'discord-line',
+					'github-fill',
+					'twitter-x-fill',
+					'discord-fill',
+					'global-fill',
+				],
 				ph: ['dot-duotone'],
 			},
 		}),
 	],
-	output: 'server',
+	output: 'hybrid',
```

Comment:
I would like to understand the motivation for this change. Can you elaborate?

Reply:
I prefer to have prerendered pages as the default when working on mostly static sites, which is our case here. I don't see the benefit of ssr eveywhere by default in our case but i may be missing something

Reply:
I generally agree, personally I would prefer to keep `output: 'server'` _(which seems to have less unexpected behavior than `hybrid`)_ and just add `export const prerender = true;` to all pages manually. But I won't die on that hill for now.